test/render_object.test.js

```diff
@@ -0,0 +1,13 @@
+import { getCategories } from '../src/helpers/render_object';
+
+describe('getCategories function', () => {
+  const container = getCategories();
+  it('Should return an HTML div tag with a class name of d-flex flex-column', () => {
+    expect(container.nodeName === 'DIV' && container.getAttribute('class')).toBe('d-flex flex-column');
+  });
+
+  it('should have category Button and collapseContainer', () => {
+    expect(container.firstChild.nodeName === 'BUTTON').toBeTruthy();
+    expect(container.lastChild.nodeName === 'DIV').toBeTruthy();
+  });
+});
```

Comment:
* [ ] The `reload` function in `reload` has been used widely but I could see where it was tested, kindly test since many functions depend on it if it fails the application fails, and to make debugging easy let test it.

Reply:
As per the video shared by Microverse and the contents we read, not everything is supposed to be tested especially when it is already going to be tested by the event listener.